package.json

```diff
@@ -85,10 +84,7 @@
       },
       "dependencies": [
         "build"
-      ],
-      "env": {
-        "DEBUG": "bidiServer:log"
-      }
```

Comment:
Interesting dilemma here. `wireit` makes the server start-up more reliable and less flaky as we can rely on `lineMatches` above to ensure the server has fully started.

OTOH, this requires `DEBUG=bidiServer:log` or `DEBUG=bidiServer:*` or `DEBUG=*` to be set. Hard-coding it here is a bad idea because then the `DEBUG` environment variable is ignored (e.g. breaks `--verbose`).

Let me try to find a compromise.

Reply:
`runBiDiServer.sh` sets `DEBUG=*` by default. Everything in this repo (including `runWPT.sh`) uses this script indirectly. This should suffice, except a few github actions.

There are two options:

1) Make the github actions use `runWPT.sh`, OR
2) Set `DEBUG=` in the github actions

Reply:
I'll go with the second approach because it's quicker. Later on we can refactor 1).